core/admin/mailu/internal/views/autoconfig.py

```diff
@@ -0,0 +1,163 @@
+from mailu.internal import internal
+
+from flask import current_app as app
+import flask
+
+@internal.route("/autoconfig/mozilla")
+def autoconfig_mozilla():
+    # https://wiki.mozilla.org/Thunderbird:Autoconfiguration:ConfigFileFormat
+    hostname = app.config['HOSTNAME']
+    xml = f'''<?xml version=\"1.0\"?>
+<clientConfig version=\"1.1\">
+<emailProvider id=\"%EMAILDOMAIN%\">
+<domain>%EMAILDOMAIN%</domain>
+
+<displayName>Email</displayName>
+<displayShortName>Email</displayShortName>
+
+<incomingServer type=\"imap\">
+<hostname>{hostname}</hostname>
+<port>993</port>
+<socketType>SSL</socketType>
+<username>%EMAILADDRESS%</username>
+<authentication>password-cleartext</authentication>
+</incomingServer>
+
+<outgoingServer type=\"smtp\">
+<hostname>{hostname}</hostname>
+<port>465</port>
+<socketType>SSL</socketType>
+<username>%EMAILADDRESS%</username>
+<authentication>password-cleartext</authentication>
+<addThisServer>true</addThisServer>
+<useGlobalPreferredServer>true</useGlobalPreferredServer>
+</outgoingServer>
+
+<documentation url=\"https://{hostname}/admin/client\">
+<descr lang=\"en\">Configure your email client</descr>
+</documentation>
+</emailProvider>
+</clientConfig>\r\n'''
+    return flask.Response(xml, mimetype='text/xml', status=200)
+
+@internal.route("/autoconfig/microsoft")
```

Comment:
Outlook receives HTTP 405 code (method not allowed) for the POST on the autodiscover URL.
The route must be changed to allow POST as method.  I suspect that this code is sufficient
Change
`@internal.route("/autoconfig/microsoft")`
to
`@internal.route("/autoconfig/microsoft", methods=['GET', 'POST'])`